public/app/plugins/datasource/cloudwatch/components/SQLBuilderEditor/utils.ts

```diff
@@ -230,6 +232,13 @@ export function setMetricName(query: CloudWatchMetricsQuery, metricName: string)
   });
 }
 
+export function removeMetricName(query: CloudWatchMetricsQuery): CloudWatchMetricsQuery {
+  const queryWithNoParams = { ...query };
+  delete queryWithNoParams.sql?.select?.parameters;
+
+  return queryWithNoParams;
+}
+
```

Comment:
This code is only tested via the SQLBuilderSelectRow tests.
Could add an explicit unit test if preferred?